### PR TITLE
Flake8 updating

### PR DESCRIPTION
**Pull request by greglucas**
Removing all of the flake8 ignores and updating the code appropriately.

All of the math still looks reasonable to me.

### CODE REVIEW

File: lib/cartopy/io/srtm.py [hunk 1]
```diff
@@ -238,12 +241,12 @@ def add_shading(elevation, azimuth, altitude):
     azimuth = np.deg2rad(azimuth)
     altitude = np.deg2rad(altitude)
     x, y = np.gradient(elevation)
-    slope = np.pi/2. - np.arctan(np.sqrt(x*x + y*y))
+    slope = np.pi / 2. - np.arctan(np.sqrt(x * x + y * y))
```
Comment (QuLogic) [inline lib/cartopy/io/srtm.py:244]:
```suggestion
    slope = np.pi / 2 - np.arctan(np.sqrt(x * x + y * y))
```

File: lib/cartopy/io/srtm.py [hunk 2]
```diff
@@ -238,12 +241,12 @@ def add_shading(elevation, azimuth, altitude):
...
     # -x here because of pixel orders in the SRTM tile
     aspect = np.arctan2(-x, y)
     shaded = np.sin(altitude) * np.sin(slope)\
         + np.cos(altitude) * np.cos(slope)\
-        * np.cos((azimuth - np.pi/2.) - aspect)
+        * np.cos((azimuth - np.pi / 2.) - aspect)
```
Comment (QuLogic) [inline lib/cartopy/io/srtm.py:249]:
```suggestion
        * np.cos((azimuth - np.pi / 2) - aspect)
```

File: lib/cartopy/tests/mpl/test_gridliner.py [hunk 3]
```diff
@@ -419,7 +419,7 @@ def test_gridliner_draw_labels_param(draw_labels, result):
     fig.canvas.draw()
     res = {}
     for loc in 'left', 'right', 'top', 'bottom':
-        artists = getattr(gl, loc+'_label_artists')
+        artists = getattr(gl, loc + '_label_artists')
```
Comment (QuLogic) [inline lib/cartopy/tests/mpl/test_gridliner.py:422]:
```suggestion
        artists = getattr(gl, f'{loc}_label_artists')
```

File: lib/cartopy/tests/mpl/test_images.py [hunk 4]
```diff
@@ -142,7 +142,7 @@ def test_imshow_wrapping():
 def test_imshow_rgba():
     # tests that the alpha of a RGBA array passed to imshow is set to 0
     # instead of masked
-    z = np.ones((100, 100))*0.5
+    z = np.ones((100, 100)) * 0.5
```
Comment (QuLogic) [inline lib/cartopy/tests/mpl/test_images.py:145]:
```suggestion
    z = np.full((100, 100), 0.5)
```

File: lib/cartopy/tests/mpl/test_images.py [hunk 5]
```diff
@@ -173,7 +174,7 @@ def test_imshow_rgba_alpha():
 def test_imshow_rgb():
     # tests that the alpha of a RGB array passed to imshow is set to 0
     # instead of masked
-    z = np.ones((100, 100, 3))*0.5
+    z = np.ones((100, 100, 3)) * 0.5
```
Comment (QuLogic) [inline lib/cartopy/tests/mpl/test_images.py:177]:
```suggestion
    z = np.full((100, 100, 3), 0.5)
```

File: lib/cartopy/tests/test_util.py [hunk 6]
```diff
@@ -293,15 +293,15 @@ def test_cyclic_has_cyclic(self):
     def test_precision_has_cyclic(self):
         '''Test precision keyword detecting cyclic point'''
         r_data = np.concatenate((self.data2d, self.data2d[:, :1]), axis=1)
-        r_lons = np.concatenate((self.lons, np.array([360.+1e-3])))
+        r_lons = np.concatenate((self.lons, np.array([360. + 1e-3])))
```
Comment (QuLogic) [inline lib/cartopy/tests/test_util.py:296]:
```suggestion
        r_lons = np.concatenate((self.lons, np.array([360 + 1e-3])))
```

File: lib/cartopy/tests/test_util.py [hunk 7]
```diff
@@ -293,15 +293,15 @@ def test_cyclic_has_cyclic(self):
...
         c_data, c_lons = add_cyclic(r_data, x=r_lons, precision=1e-2)
         assert_array_equal(c_data, r_data)
         assert_array_equal(c_lons, r_lons)
 
     def test_precision_has_cyclic_no(self):
         '''Test precision keyword detecting no cyclic point'''
         new_data = np.concatenate((self.data2d, self.data2d[:, :1]), axis=1)
-        new_lons = np.concatenate((self.lons, np.array([360.+1e-3])))
+        new_lons = np.concatenate((self.lons, np.array([360. + 1e-3])))
```
Comment (QuLogic) [inline lib/cartopy/tests/test_util.py:304]:
```suggestion
        new_lons = np.concatenate((self.lons, np.array([360 + 1e-3])))
```

File: lib/cartopy/tests/test_util.py [hunk 8]
```diff
@@ -411,6 +411,6 @@ def test_3d_axis_cyclic(self):
 
     def test_1d_precision(self):
         '''Test 1d with precision keyword'''
-        new_clons = np.concatenate((self.lons, np.array([360.+1e-3])))
+        new_clons = np.concatenate((self.lons, np.array([360. + 1e-3])))
```
Comment (QuLogic) [inline lib/cartopy/tests/test_util.py:414]:
```suggestion
        new_clons = np.concatenate((self.lons, np.array([360 + 1e-3])))
```

**Review by QuLogic — COMMENTED** [review]
LGTM; some slight cleanup can be done:

**Comment by greglucas** [comment]
Thanks got those and added a few more from the `test_utils`.

**Review by QuLogic — APPROVED** [review]

**Pull request merged**